source/manuals/logit-io.html.md.erb

```diff
@@ -4,172 +4,138 @@ title: Sending logs securely to Logit.io
 
-TCP ports are assigned on a per-stack basis at stack creation time and will be
-different per stack. We assume this is so that Logit can share IP addresses
-between many stacks.
+TCP ports are assigned on a per-stack basis at stack creation time and will be different per stack.
 
 ## Shut down unsecure inputs
 
-Logit stacks have a very lax security configuration by default. Each of the
-inputs listed will be behind a TCP port that is open to the internet at large,
-and will not authenticate clients that connect.
+Logit stacks have a non-restrictive security configuration by default. Each of the inputs listed will be behind a TCP port that is open to the internet at large, and will not authenticate connecting clients.
 
-We are interested only in `Beats-SSL` and `Syslog-SSL`. Put in a support request
-to Logit to close the TCP or UDP ports for all other protocols.
+The only inputs we are interested in are **Beats-SSL** and **Syslog-SSL**, you will need to log a support request with Logit to close the TCP or UDP ports for all other protocols.
 
-If you know in advance which IP addresses or address ranges you will be sourcing
-logging traffic from, include these in your request so that the `Beats-SSL` and
-`Syslog-SSL` ports can be whitelisted.
+If you know in advance which IP addresses or address ranges you will be sourcing logging traffic from, include these in your request so that the **Beats-SSL** and **Syslog-SSL** ports can be whitelisted.
 
-## Request TLS mutual auth
+## Request TLS mutual authentication
 
-Also via a support request to Logit, request that TLS mutual auth be switched on
-for your stack.
+Request TLS mutual authentication is switched on for your stack. You can do this through a support request to Logit.
 
-Logit provide two methods by which this can be done. With one method, we provide
-them with a CA certificate of our own and instruct them to trust client certificates
-signed by that CA for access to the stack.
+Logit provides two methods to achieve this:
 
-The other, simpler method involves Logit generating a client private key and
-certificate on our behalf, signing it with their CA, and sending it to us.
+* provide Logit with a CA certificate of your own and instruct them to trust client certificates signed by that CA for access to the stack
+
+* the second, simpler method involves Logit generating a client private key and certificate on your behalf, signing it with their CA and sending it back
 
 This manual will cover just the second, simpler method.
 
 ## Logit PKI architecture
 
-If you were to inspect the certificate presented by Logit's TLS endpoints on a
-freshly provisioned stack with the OpenSSL command line tool...
+You can inspect the certificate presented by Logit's TLS endpoints on a
+freshly provisioned stack using the OpenSSL command line tool:
 
 ```
 $ openssl s_client -connect f046ca39-8388-4c49-a536-19f95a555905-ls.logit.io:12345 </dev/null
```

Comment:
Maybe this would be better as

```
 $ openssl s_client -connect <stack-id>-ls.logit.io:<assigned-port> 
```

I got a bit confused when I replaced the stack-id but left it with port 12345 not noticing that wasn't a standard syslog port but a placeholder for the assigned port from logit (I guess they can't use something like Server Name Indication on syslog).